code_sponsor/settings.py

```diff
@@ -166,5 +161,26 @@
 CELERY_ACCEPT_CONTENT = ["json"]
 CELERY_RESULT_BACKEND = None
 
-MAILGUN_SENDING_DOMAIN = env('MAILGUN_SENDING_DOMAIN', default='False')
-MAILGUN_API_KEY = env('MAILGUN_API_KEY', default='False')
+# Email config
+ANYMAIL = {
+    "MAILGUN_API_KEY": env('MAILGUN_API_KEY', default='False'),
```

Comment:
I recommend using either `''` or `False` for the default here.  `'False'` might get confusing.